.vortex/CLAUDE.md

```
@@ -89,6 +99,31 @@ run_steps "assert" "${mocks}"   # Assert phase
 
 ### Running Tests
 
+**Documentation Tests** (`.vortex/docs/`):
+```bash
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines
null

(MD031, blanks-around-fences)